### ISSUE TITLE
OneLogin::RubySaml::SettingError: Invalid settings, idp_slo_service_url is not set!

### ISSUE THREAD

**Issue by anwarNazig** [apokalipto/devise_saml_authenticatable#198]
Hi following the googleApp wiki  https://github.com/apokalipto/devise_saml_authenticatable/wiki/Google-Apps i found that:
settings.idp_slo_target_url is set to empty. i've seen that instead of target we should use service as shown below: 

> Config.saml_configure do |settings|
>       settings.idp_slo_service_url  = ""
> end
The problem is when i try to logout fro mthe application, i get that  settings.idp_slo_service_url  must be set:

> OneLogin::RubySaml::SettingError: Invalid settings, idp_slo_service_url is not set!

the problem is duo to the fact that i want to logout from my application (ISP) and redirect to the google login page when i sign_out .

**Comment by adamstegman** [comment]
Great find, that empty value looks incorrect. Is there a URL that Google gives you for this setting you can use?

**Comment by anwarNazig** [comment]
No Google does not give any url for this setting  due to the fact that they does not support single sign out. do you have any suggestion on how to solve the issue, or at least a workaround for the moment?

**Comment by adamstegman** [comment]
Yes, you can override `after_sign_out_path_for` in the controller to send them somewhere else, like back to Google or to an unauthenticated page in your site.

In an initializer, try adding:

```ruby
Devise::SamlSessionsController.class_eval do
  def after_sign_out_path_for(_)
    root_path
  end
end
```
